coreTable/CoreTableColumns.ts

```diff
@@ -191,7 +191,7 @@ abstract class AbstractCoreColumn<JS_TYPE extends PrimitiveType> {
 
     // A method for formatting for CSV
     formatForCsv(value: JS_TYPE): string {
-        return csvEscape(this.formatValue(value))
```

Comment:
add a test? hard to tell what the broke behavior was, and what the new behavior is.

Reply:
Right, makes sense! will do that.